css/style.css

```diff
@@ -541,3 +543,75 @@ nav {
     width: 100%;
   }
 }
+
+.modal-container {
+  display: none;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.6);
+  inset: 0 0;
+  width: 100vw;
+  height: 100vh;
+  padding: 10% 0;
+  margin: 0;
+}
+
+.modal {
+  background-color: #fff;
+  width: 90vw;
+  height: auto;
+  padding: 5%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 16px;
+}
```

Comment:
- [ ] Please make sure the project description is aligned at the project's image start point and model buttons are aligned at the project's image end point and please make sure it matches with the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168033561-c5f25b17-2626-45cc-b270-14c7823a77f7.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168033594-ce05649a-2dab-40ab-af13-5a2767fe9291.png)